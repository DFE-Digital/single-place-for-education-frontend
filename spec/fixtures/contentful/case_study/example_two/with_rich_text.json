{
  "sys": {
    "type": "Array"
  },
  "total": 1,
  "skip": 0,
  "limit": 100,
  "items": [
    {
      "sys": {
        "space": {
          "sys": {
            "type": "Link",
            "linkType": "Space",
            "id": "gfzscr0ovdsz"
          }
        },
        "id": "5qpbGnRDJfFW8f0DZU0qkV",
        "type": "Entry",
        "createdAt": "2019-06-25T14:14:57.408Z",
        "updatedAt": "2019-06-25T14:15:24.005Z",
        "environment": {
          "sys": {
            "id": "master",
            "type": "Link",
            "linkType": "Environment"
          }
        },
        "revision": 2,
        "contentType": {
          "sys": {
            "type": "Link",
            "linkType": "ContentType",
            "id": "caseStudy"
          }
        },
        "locale": "en-US"
      },
      "fields": {
        "name": "Case Study - Woof Primary School",
        "slug": "woof-primary-school-case-study",
        "heroImage": {
          "sys": {
            "type": "Link",
            "linkType": "Entry",
            "id": "4NSN5Mw0UjgVqs8KObVyde"
          }
        },
        "content": [
          {
            "sys": {
              "type": "Link",
              "linkType": "Entry",
              "id": "1KrUMiG0OBiyHHhDaejbRv"
            }
          }
        ]
      }
    }
  ],
  "includes": {
    "Entry": [
      {
        "sys": {
          "space": {
            "sys": {
              "type": "Link",
              "linkType": "Space",
              "id": "gfzscr0ovdsz"
            }
          },
          "id": "1KrUMiG0OBiyHHhDaejbRv",
          "type": "Entry",
          "createdAt": "2019-06-25T14:14:27.196Z",
          "updatedAt": "2019-06-25T14:43:36.483Z",
          "environment": {
            "sys": {
              "id": "master",
              "type": "Link",
              "linkType": "Environment"
            }
          },
          "revision": 8,
          "contentType": {
            "sys": {
              "type": "Link",
              "linkType": "ContentType",
              "id": "richText"
            }
          },
          "locale": "en-US"
        },
        "fields": {
          "title": "Woof Rich Text",
          "content": {
            "nodeType": "document",
            "data": {},
            "content": [
              {
                "nodeType": "heading-1",
                "content": [
                  {
                    "nodeType": "text",
                    "value": "Woof Heading 1",
                    "marks": [],
                    "data": {}
                  }
                ],
                "data": {}
              },
              {
                "nodeType": "heading-2",
                "content": [
                  {
                    "nodeType": "text",
                    "value": "Woof Heading 2",
                    "marks": [],
                    "data": {}
                  }
                ],
                "data": {}
              },
              {
                "nodeType": "heading-3",
                "content": [
                  {
                    "nodeType": "text",
                    "value": "Woof Heading 3",
                    "marks": [],
                    "data": {}
                  }
                ],
                "data": {}
              },
              {
                "nodeType": "heading-4",
                "content": [
                  {
                    "nodeType": "text",
                    "value": "Woof Heading 4",
                    "marks": [],
                    "data": {}
                  }
                ],
                "data": {}
              },
              {
                "nodeType": "paragraph",
                "content": [
                  {
                    "nodeType": "text",
                    "value": "Woof. Woof. Woof. This is normal text.",
                    "marks": [],
                    "data": {}
                  }
                ],
                "data": {}
              },
              {
                "nodeType": "paragraph",
                "content": [
                  {
                    "nodeType": "text",
                    "value": "Woof. Woof. Woof. This is bold text.",
                    "marks": [
                      {
                        "type": "bold"
                      }
                    ],
                    "data": {}
                  }
                ],
                "data": {}
              },
              {
                "nodeType": "paragraph",
                "content": [
                  {
                    "nodeType": "text",
                    "value": "Woof. Woof. Woof. This is italic text.",
                    "marks": [
                      {
                        "type": "italic"
                      }
                    ],
                    "data": {}
                  }
                ],
                "data": {}
              },
              {
                "nodeType": "paragraph",
                "content": [
                  {
                    "nodeType": "text",
                    "value": "Woof. Woof. Woof. This is underlined text.",
                    "marks": [
                      {
                        "type": "underline"
                      }
                    ],
                    "data": {}
                  }
                ],
                "data": {}
              },
              {
                "nodeType": "paragraph",
                "content": [
                  {
                    "nodeType": "text",
                    "value": "",
                    "marks": [],
                    "data": {}
                  },
                  {
                    "nodeType": "hyperlink",
                    "content": [
                      {
                        "nodeType": "text",
                        "value": "Woof. Woof. Woof. This is a link.",
                        "marks": [],
                        "data": {}
                      }
                    ],
                    "data": {
                      "uri": "https://woof.dog"
                    }
                  },
                  {
                    "nodeType": "text",
                    "value": "",
                    "marks": [],
                    "data": {}
                  }
                ],
                "data": {}
              },
              {
                "nodeType": "unordered-list",
                "content": [
                  {
                    "nodeType": "list-item",
                    "data": {},
                    "content": [
                      {
                        "nodeType": "paragraph",
                        "content": [
                          {
                            "nodeType": "text",
                            "value": "Woof. Woof. Woof. This is an unordered list item 1.",
                            "marks": [],
                            "data": {}
                          }
                        ],
                        "data": {}
                      }
                    ]
                  },
                  {
                    "nodeType": "list-item",
                    "content": [
                      {
                        "nodeType": "paragraph",
                        "content": [
                          {
                            "nodeType": "text",
                            "value": "Woof. Woof. Woof. This is an unordered list item 2.",
                            "marks": [],
                            "data": {}
                          }
                        ],
                        "data": {}
                      }
                    ],
                    "data": {}
                  },
                  {
                    "nodeType": "list-item",
                    "content": [
                      {
                        "nodeType": "paragraph",
                        "content": [
                          {
                            "nodeType": "text",
                            "value": "Woof. Woof. Woof. This is an unordered list item 3.",
                            "marks": [],
                            "data": {}
                          }
                        ],
                        "data": {}
                      }
                    ],
                    "data": {}
                  }
                ],
                "data": {}
              },
              {
                "nodeType": "ordered-list",
                "content": [
                  {
                    "nodeType": "list-item",
                    "data": {},
                    "content": [
                      {
                        "nodeType": "paragraph",
                        "content": [
                          {
                            "nodeType": "text",
                            "value": "Woof. Woof. Woof. This is an ordered list item 1.",
                            "marks": [],
                            "data": {}
                          }
                        ],
                        "data": {}
                      }
                    ]
                  },
                  {
                    "nodeType": "list-item",
                    "content": [
                      {
                        "nodeType": "paragraph",
                        "content": [
                          {
                            "nodeType": "text",
                            "value": "Woof. Woof. Woof. This is an ordered list item 2.",
                            "marks": [],
                            "data": {}
                          }
                        ],
                        "data": {}
                      }
                    ],
                    "data": {}
                  },
                  {
                    "nodeType": "list-item",
                    "content": [
                      {
                        "nodeType": "paragraph",
                        "content": [
                          {
                            "nodeType": "text",
                            "value": "Woof. Woof. Woof. This is an ordered list item 3.",
                            "marks": [],
                            "data": {}
                          }
                        ],
                        "data": {}
                      }
                    ],
                    "data": {}
                  }
                ],
                "data": {}
              },
              {
                "nodeType": "paragraph",
                "content": [
                  {
                    "nodeType": "text",
                    "data": {},
                    "value": "",
                    "marks": []
                  }
                ],
                "data": {}
              }
            ]
          }
        }
      },
      {
        "sys": {
          "space": {
            "sys": {
              "type": "Link",
              "linkType": "Space",
              "id": "gfzscr0ovdsz"
            }
          },
          "id": "4NSN5Mw0UjgVqs8KObVyde",
          "type": "Entry",
          "createdAt": "2019-06-20T14:14:59.261Z",
          "updatedAt": "2019-06-20T14:14:59.261Z",
          "environment": {
            "sys": {
              "id": "master",
              "type": "Link",
              "linkType": "Environment"
            }
          },
          "revision": 1,
          "contentType": {
            "sys": {
              "type": "Link",
              "linkType": "ContentType",
              "id": "heroImage"
            }
          },
          "locale": "en-US"
        },
        "fields": {
          "name": "Happy dog in meeting",
          "file": {
            "sys": {
              "type": "Link",
              "linkType": "Asset",
              "id": "5t6LH0f5htWyTQ8YdpIYXz"
            }
          }
        }
      }
    ],
    "Asset": [
      {
        "sys": {
          "space": {
            "sys": {
              "type": "Link",
              "linkType": "Space",
              "id": "gfzscr0ovdsz"
            }
          },
          "id": "5t6LH0f5htWyTQ8YdpIYXz",
          "type": "Asset",
          "createdAt": "2019-06-20T14:14:39.328Z",
          "updatedAt": "2019-06-20T14:14:56.444Z",
          "environment": {
            "sys": {
              "id": "master",
              "type": "Link",
              "linkType": "Environment"
            }
          },
          "revision": 2,
          "locale": "en-US"
        },
        "fields": {
          "title": "happy-dog-in-meeting",
          "description": "Happy dog in meeting",
          "file": {
            "url": "https://some-image-somewhere-else",
            "details": {
              "size": 1303022,
              "image": {
                "width": 1920,
                "height": 1280
              }
            },
            "fileName": "happy-dog-in-meeting.jpg",
            "contentType": "image/jpeg"
          }
        }
      }
    ]
  }
}
