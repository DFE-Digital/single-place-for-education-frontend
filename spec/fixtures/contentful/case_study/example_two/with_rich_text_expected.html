<h1 class="govuk-heading-xl" id="woof-heading-1">Woof Heading 1</h1>
<h2 class="govuk-heading-l" id="woof-heading-2">Woof Heading 2</h2>
<h3 class="govuk-heading-m" id="woof-heading-3">Woof Heading 3</h3>
<h4 class="govuk-heading-s" id="woof-heading-4">Woof Heading 4</h4>
<p class="govuk-body">Woof. Woof. Woof. This is normal text.</p>
<p class="govuk-body"><b>Woof. Woof. Woof. This is bold text.</b></p>
<p class="govuk-body"><i>Woof. Woof. Woof. This is italic text.</i></p>
<p class="govuk-body"><u>Woof. Woof. Woof. This is underlined text.</u></p>
<p class="govuk-body"><a class="govuk-link" href="https://woof.dog">Woof. Woof. Woof. This is a link.</a></p>
<ul class="govuk-list govuk-list--bullet"><li><p class="govuk-body">Woof. Woof. Woof. This is an unordered list item 1.</p></li><li><p class="govuk-body">Woof. Woof. Woof. This is an unordered list item 2.</p></li><li><p class="govuk-body">Woof. Woof. Woof. This is an unordered list item 3.</p></li></ul>
<ol class="govuk-list govuk-list--number"><li><p class="govuk-body">Woof. Woof. Woof. This is an ordered list item 1.</p></li><li><p class="govuk-body">Woof. Woof. Woof. This is an ordered list item 2.</p></li><li><p class="govuk-body">Woof. Woof. Woof. This is an ordered list item 3.</p></li></ol>
<p class="govuk-body"></p>
